lib/puppet/gettext/config.rb

```diff
@@ -5,6 +5,14 @@ module Puppet::GettextConfig
   POSIX_PATH = File.absolute_path('../../../../../share/locale', File.dirname(__FILE__))
   WINDOWS_PATH = File.absolute_path('../../../../../../../puppet/share/locale', File.dirname(__FILE__))
 
+  begin
+    require 'gettext-setup'
+    require 'locale'
+    @found = true
```

Comment:
defining an instance variable on a module this way doesn't seem to work for me:
```ruby
[7] pry(main)> module Foo
[7] pry(main)*   @found = true
[7] pry(main)*   def what_is_found
[7] pry(main)*     puts "found is #{@found}"
[7] pry(main)*   end
[7] pry(main)* end
=> :what_is_found
[8] pry(main)> class Bar
[8] pry(main)*   include Foo
[8] pry(main)* end
=> Bar
[9] pry(main)> baz = Bar.new
=> #<Bar:0x007fadb1ceb420>
[10] pry(main)> baz.what_is_found
found is
=> nil
```

However, if I set the value via an included method, it seems to work:
```ruby
[11] pry(main)> module Foo
[11] pry(main)*   def set_found(found)
[11] pry(main)*     @found = found
[11] pry(main)*   end
[11] pry(main)*   def what_is_found
[11] pry(main)*     @found
[11] pry(main)*   end
[11] pry(main)* end
=> :what_is_found
[12] pry(main)> class Bar
[12] pry(main)*   include Foo
[12] pry(main)* end
=> Bar
[13] pry(main)> baz = Bar.new
=> #<Bar:0x007fadb22b11e8>
[14] pry(main)> baz.what_is_found
=> nil
[15] pry(main)> baz.set_found("10")
=> "10"
[16] pry(main)> baz.what_is_found
=> "10"
```

Reply:
This seems to work... going to dig into it a bit. Maybe translation is never loaded now.

Reply:
Ah, you're missing that it's defining `self.what_is_found`.

Reply:
I was stepping through this with pry earlier, and at least internal to the module, `@found` does seem to keep its value. Not sure about the include case, but we (at least currently) do not use this module that way.

Reply:
Which we then access directly via `Foo.what_is_found`.

Reply:
Ah you're right, I see. We're using class methods inside this module rather than instance methods we mix in.
```ruby
[1] pry(main)> module Foo
[1] pry(main)*   @found = true
[1] pry(main)*   def self.what_is_found
[1] pry(main)*     puts "found is #{@found}"
[1] pry(main)*   end
[1] pry(main)* end
=> :what_is_found
[2] pry(main)> Foo.what_is_found
found is true
=> nil
```

works